.travis.yml

```diff
@@ -1,13 +1,32 @@
 language: ruby
 cache: bundler
 script: bundle exec middleman build
+
+env:
+  global:
+  # CloudFoundry credentials for deployment
+  - CF_API="https://api.cloud.service.gov.uk"
+  - CF_ORG="govuk-design-system"
+  - CF_SPACE="prototype-kit-training"
+  - CF_USERNAME="prototype-kit-docs-deploy-production@digital.cabinet-office.gov.uk"
```

Comment:
Did you setup a new email or did this always exist? is this in the credentials repo?

Reply:
https://github.com/alphagov/design-system-team-credentials/tree/master/paas/prototype-kit-docs

Reply:
It's been added to the credentials repo as part of this work 👍